openmdao/vectors/vector.py

```diff
@@ -620,3 +561,23 @@ def print_variables(self):
             print(' ' * 3, prom_name, view)
         print('-' * 35)
         print()
+
+    def set_complex_step_mode(self, flag):
+        """
+        Turn on or off complex stepping mode.
+
+        When turned on, the default real ndarray is replaced with a complex ndarray and all
+        pointers are updated to point to it.
+
+        Parameters
+        ----------
+        flag : bool
```

Comment:
I would suggest changing 'flag' to something a little less generic, maybe 'active' or something like that.